lib/webdriveragent.js

```diff
@@ -287,6 +288,73 @@ class WebDriverAgent {
     }
   }
 
+
+  /**
+   * @typedef {Object} LaunchingEnvironmentOptions
+   * @property {number} USE_PORT
+   * @property {string} WDA_PRODUCT_BUNDLE_IDENTIFIER
+   */
+
+  /**
+   * Launch WDA with preinstalled package with 'xcrun devicectl device process launch'.
+   * The WDA package must be prepared properly like published via
+   * https://github.com/appium/WebDriverAgent/releases
+   * with proper sign for this case.
+   *
+   * When we implement launching XCTest service via appium-ios-device,
+   * this implementation can be replaced with it.
+   *
+   * @param {LaunchingEnvironmentOptions}  launchingEnv launching environment for WebDriverAgent.
+   * @return {Promise<void>}
+   */
+
+  async _launchWdaViaDeviceCtl(launchingEnv) {
```

Comment:
lets just have generic options, where env is a member, so we could add more items later, like `{env: ...}`